content/numpy/concepts/math-methods/terms/trunc/trunc.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: '.trunc()'
+Description: 'Truncate the decimal part of each element in an array, returning the integer part as a float.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+  - 'Formatting'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+**`numpy.trunc()`** truncates the decimal part of each element in a NumPy array, returning the integer part of the number as a float. It is often used when you need to ignore the fractional part without rounding.
+
+## Syntax
+
+```pseudo
+numpy.trunc(x, /, out=None, *, where=True)
+```
+
+- `x`: The input array whose elements you want to truncate.
+- `out`: (Optional) A location where the result is stored. If not provided, a new array is created.
+- `where`: (Optional) A boolean array that specifies which elements to compute for. Default is True for all elements.
+
+It returns a NumPy array containing the truncated values, where each element is the integer part of the input.
+
+## Example
+
+### Example 1: Truncating Decimal Values in an Array
```

Comment:
```suggestion
```

Reply:
The heading can be added to the description itself. For examples we need h2 heading:
##Example 1
##Example 2 and so on